admin/app/components/organization-information-section.js

```diff
@@ -1,8 +1,13 @@
 import { action } from '@ember/object';
+import ENV from 'pix-admin/config/environment';
 import Component from '@glimmer/component';
 
 export default class OrganizationInformationSection extends Component {
 
+  get externalURL() {
```

Comment:
C'est pas très précis `externalURL`, ça pourrait aussi bien désigner l'URL du site de l'organisation en question non ?
Je propose
```suggestion
  get externalDashboardURL() {
```